CHANGELOG.md

```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [0.3.0] - 2021-04-09
```

Comment:
Why have only fixes here. Why is it a minor bump?

Reply:
@kopiczko just forget you've seen this PR 😄